peredelano-profile/package.json

```diff
@@ -1,17 +1,25 @@
 {
   "name": "peredelano-profile",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "type": "module",
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview"
   },
-  "files": ["dist/**"],
+  "files": [
+    "dist/**"
+  ],
   "dependencies": {
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.11.16",
+    "@mui/material": "^5.13.2",
+    "@mui/styled-engine-sc": "^5.12.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "styled-components": "^5.3.11"
```

Comment:
We don't plan to use `styled-components` yet. Most likely we will use PostCSS. Call `yarn remove styled-components @mui/styled-engine-sc`